### DISCUSSION TITLE
Add an ability to specify the attacker in AbilityEffectType.OnAttacked

### DISCUSSION THREAD

**Discussion by cabegame** [FakeFishGames/Barotrauma#15376] [Ideas and suggestions]
It would be great for modders to have an ability to specify the attacker in their talents
Like trigger this thing when attacked by a monster or do that when hit by a human enemy
Right now it's triggered by everyone

**Comment by A2ur3** [comment]
This is so stupid. No matter if I set targetabilitytarget to true or false, it always points to the same target.
<img width="1090" height="693" alt="image" src="https://github.com/user-attachments/assets/d370eb03-2222-4d7f-8897-bc9eca836cde" />
